src/frontend/src/binder/relation/window_table_function.rs

```diff
@@ -93,46 +93,17 @@ impl Binder {
             .into());
         };
 
-        self.pop_context();
-
-        let table_catalog =
-            self.catalog
-                .get_table_by_name(&self.db_name, &schema_name, &table_name)?;
+        let base_columns = std::mem::take(&mut self.context.columns);
 
-        let columns = table_catalog.columns().to_vec();
-        if columns.iter().any(|col| {
-            col.name().eq_ignore_ascii_case("window_start")
-                || col.name().eq_ignore_ascii_case("window_end")
-        }) {
-            return Err(ErrorCode::BindError(
-                "column names `window_start` and `window_end` are not allowed in window table function's input."
-                .into())
-            .into());
```

Comment:
The error is removed?

Reply:
In fact the error here didn't make sense and I was going to raise an issue for that. Because that will mean we do not support tvf-on-tvf. 

For example, HOP(select * from tumble(t, report_time), window_end),

Reply:
But how does the user specify which `window_start` column to use in this case?

Reply:
What’s the use case of TVF over TVF?

Reply:
Add check back now, since it's hard to handle the case.